app/views/transactions/_transaction.html.erb

```diff
@@ -1,17 +1,20 @@
+<%# locals: (transaction:, is_widget: false) %>
```

Comment:
Small nitpick here, but since we're unsure _where_ a transaction will be shown in the UI, do you think instead of `is_widget`, we introduce something like `fields: [:name, :category, :account, :amount]` that explicitly tells the partial which transaction fields to display and the sizing automatically works based on that?

Reply:
I think introducing the `fields` concept is a good idea! However, with the current code structure, dynamic size calculation could be tricky. I'm not entirely sure how to handle it because the same partial should fit both a narrow widget and a wide page while keeping the columns aligned.

Using a `<table>` could potentially fix this issue, but it might be a bit old-fashioned

https://github.com/maybe-finance/maybe/blob/c46662c99f68fd1deab5680b0dd4deb31a20eda8/app/views/transactions/_transaction.html.erb#L3
https://github.com/maybe-finance/maybe/blob/c46662c99f68fd1deab5680b0dd4deb31a20eda8/app/views/transactions/_transaction.html.erb#L8

Reply:
If it's too challenging I think it's okay if we keep it as-is.  A table will introduce styling challenges of its own and make it less flexible to deal with if we start introducing inline turbo frame editing to this partial.